packages/vats/src/core/chain-behaviors.js

```diff
@@ -460,6 +460,7 @@ export const publishAgoricNames = async (
   const marshaller = E(board).getPublishingMarshaller();
 
   // XXX will fail upon restart, but so would the makeStoredPublishKit this is replacing
+  // XXX is that ok?
```

Comment:
yes, there's only one production use of `makeStoredPublishKit` which is  [vote outcomes](https://github.com/Agoric/agoric-sdk/blob/bb30947461eae614ff5e4fac3a9e21b927efe3f6/packages/governance/src/committee.js#L142-L147) and we've elected to defer durability of governance.

Reply:
Updated with comment explaining why this is ok